CHANGELOG.md

```diff
@@ -20,6 +22,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - User experience for situations where Conjur identity cannot be resolved has been
   improved with better errors/warnings.
   [cyberark/conjur-puppet#125](https://github.com/cyberark/conjur-puppet/issues/125).
+- Added warnings and better error logic to certificate parsing logic.
+  [cyberark/conjur-puppet#115](https://github.com/cyberark/conjur-puppet/issues/115).
+
+### Fixed
+- Fixed account name escaping during variable fetching.
+  [cyberark/conjur-puppet#170](https://github.com/cyberark/conjur-puppet/issues/170).
 
 ### Removed
 - Support for using the Conjur Puppet module with Conjur Enterprise v4 is removed
```

Comment:
```suggestion
- Support for using the Conjur Puppet module with Conjur Enterprise v4.
```
"is removed" is implied by the heading

also, the period after the link can be removed everywhere in this changelog section